tests/unit/FilingHistoryList1.spec.ts

```diff
@@ -772,7 +772,7 @@ describe('Filing History List - redirections', () => {
     business: { identifier: 'CP1002587', legalName: undefined, legalType: 'CP' },
     correction: {
       comment: '',
-      correctedFilingDate: 'Wed, 24 May 2023 18:02:16 GMT',
```

Comment:
This and line 885 were the wrong format, but the tests didn't catch this because we're mocking the API response (and it's the API that complains about the wrong format.